settings.gradle

```diff
@@ -26,3 +26,7 @@ include 'TabHostExample'
 include 'TabHostSelfContainedTabBrowsing'
 include 'wizard'
 include 'EditTextChips'
+include 'WearBuildConfigMobile'
+project(':WearBuildConfigMobile').projectDir = new File('WearBuildConfig/mobile')
```

Comment:
ok, I think I see what you're trying to do, but I'm still not sure why you need to create this alias (and the one below). If you're trying to workaround the multi module setup in the whole `android-demos` maybe this is not the best solution. clarifying your intent here would help a lot :)